certified-connectors/OpenLegacy IBM I (AS400)/apiDefinition.swagger.json

```diff
@@ -1,318 +1,477 @@
-                    {
-                        "name": "enrichmentType",
-                        "in": "query",
-                        "default": "HTTP",
-                        "type": "string",
-                        "x-ms-visibility": "internal",
-                        "x-ms-summary": "Determines the type of enrichment metadata to be included in the response.",
-                        "description": "Determines the type of enrichment metadata to be included in the response.",
-                        "required": true
-                    }
-                ],
-                "x-ms-visibility": "internal",
-                "responses": {
-                    "200": {
-                        "description": "OK",
-                        "schema": {
-                            "$ref": "#/definitions/GetMethodsForContractResponse"
-                        }
-                    }
-                }
-            }
-        },
-        "/api/v1/partners/method-swagger-by-id/{methodId}": {
-            "get": {
-                "summary": "Get Method Open Api Spec",
-                "operationId": "GetMethodOpenApiSpec",
-                "description": "Get a method OpenApi specification",
-                "parameters": [
-                    {
-                        "name": "methodId",
-                        "in": "path",
-                        "required": true,
-                        "type": "string",
-                        "x-ms-url-encoding": "single",
-                        "x-ms-summary": "Method ID for which an OpenApi specification must be returned.",
-                        "description": "Method ID for which an OpenApi specification must be returned."
-                    }
-                ],
-                "x-ms-visibility": "internal",
-                "responses": {
-                    "200": {
-                        "description": "OK",
-                        "schema": {
-                            "type": "object",
-                            "properties": {
-                                "flow-input": {
-                                    "type": "object"
-                                },
-                                "flow-output": {
-                                    "type": "object"
-                                }
-                            }
-                        }
-                    }
-                }
-            }
+  "swagger": "2.0",
+  "info": {
+      "title": "OpenLegacy IBM I (AS400)",
+      "description": "OpenLegacy IBM i (AS400) connector provides bidirectional connectivity to your system’s data and business logic through APIs deployed on OpenLegacy’s cloud or any other target deployment. You can now build Business Flows with information from your legacy environment using the OpenLegacy connector. This will enable you to utilize your legacy assets as digital components, providing you a legacy integration and modernization solution from Day 1.",
```

Comment:
@boiv please remove or replace the word "API" in the description. please see [here ](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info).

Reply:
@Amjed-Ayoub 
The description was updated